uvicorn/protocols/utils.py

```diff
@@ -14,8 +15,12 @@ def get_remote_addr(transport):
         else:
             family = socket_info.family
 
-        if family in (socket.AF_INET, socket.AF_INET6):
-            return (str(info[0]), int(info[1]))
+        if not sys.platform.startswith("win"):
```

Comment:
Why do we need such check? What happens when `socket.AF_UNIX` is tested on windows? Or its undefined on win?

Reply:
you'd get an error :
`AttributeError: module 'socket' has no attribute 'AF_UNIX'`
see https://travis-ci.org/github/euri10/uvicorn/jobs/674739989#L647